index.bs

```diff
@@ -364,79 +364,80 @@ deletion are considered to be the responsibility of such a user interface and ar
+    :   {{PublicKeyCredential/rawId}}
+    ::  This attribute returns the {{ArrayBuffer}} contained in the {{PublicKeyCredential/[[identifier]]}} internal slot.
 
     :   <dfn>response</dfn>
-    ::  This attribute contains the [=authenticator=]'s response to the client's request to either create a [=scoped
-        credential=], or generate an [=authentication assertion=]. If the {{ScopedCredential}} is created in response to
+    ::  This attribute contains the [=authenticator=]'s response to the client's request to either create a [=public key
+        credential=], or generate an [=authentication assertion=]. If the {{PublicKeyCredential}} is created in response to
         {{CredentialsContainer/create()}}, this attribute's value will be an {{AuthenticatorAttestationResponse}}, otherwise,
-        the {{ScopedCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
+        the {{PublicKeyCredential}} was created in response to {{CredentialsContainer/get()}}, and this attribute's value
         will be an {{AuthenticatorAssertionResponse}}.
 
     :   <dfn>clientExtensionResults</dfn>
     ::  This attribute contains a [=map=] containing [=extension identifier=] → [=client extension output=] entries
         produced by the extension's [=client extension processing=].
 
     :   <dfn>\[[type]]</dfn>
-    ::  The {{ScopedCredential}} [=interface object=]'s {{Credential/[[type]]}} [=internal slot=]'s value is the string
-        "`scoped`".
+    ::  The {{PublicKeyCredential}} [=interface object=]'s {{Credential/[[type]]}} [=internal slot=]'s value is the string
+        "`publickey`".
 
         Note: This is reflected via the {{Credential/type}} attribute getter inherited from {{Credential}}.
 
     :   {{Credential/[[discovery]]}}
-    ::  The {{ScopedCredential}} [=interface object=]'s {{Credential/[[discovery]]}} [=internal slot=]'s value is
+    ::  The {{PublicKeyCredential}} [=interface object=]'s {{Credential/[[discovery]]}} [=internal slot=]'s value is
         "{{Credential/[[discovery]]/remote}}".
 
     :   <dfn>\[[identifier]]</dfn>
-    ::  This [=internal slot=] contains an identifier for the credential, chosen by the platform with help from the authenticator.
-        This identifier is used to look up credentials for use, and is therefore expected to be globally unique with high
-        probability across all credentials of the same type, across all authenticators. This API does not constrain the format
-        or length of this identifier, except that it must be sufficient for the platform to uniquely select a key. For example,
-        an authenticator without on-board storage may create identifiers containing a [=credential private key=] wrapped with a
-        symmetric key that is burned into the authenticator.
+    ::  This [=internal slot=] contains an identifier for the credential, chosen by the platform with help from the
```

Comment:
I think the file is mostly wrapped to 130 characters. To avoid churn, maybe don't rewrap to less than that when you're not otherwise changing the paragraph?